java/com/scrat/everchanging/EverchangingRender.java

```diff
@@ -200,11 +199,12 @@ public void onSurfaceChanged(final GL10 gl, final int width, final int height) {
         GLES20.glViewport(0, 0, width, height);
 
         final float scaleImageHeight = surfaceHeight / 320f;
+        final float scaleImageWidth = 240f / width;
         //todo setup position
         final int scenesSize = scenes.size();
         for (int i = 0; i < scenesSize; i++) {
             scenes.get(i)
-                    .setupPosition(surfaceWidth, surfaceHeight, scaleImageHeight, displayRotation);
+                    .setupPosition(surfaceWidth, surfaceHeight, (i<scenesSize-1)?scaleImageHeight:scaleImageWidth, displayRotation);

```

Comment:
Sorry if I broke this. This code is not formatted now BTW (no spaces between operators).
Usually when I make any changes I press IDE shortcut to run a formatter.
Please consider doing the same for consistent formatting.

Reply:
See https://trunk.io/learn/what-is-a-code-formatter-and-why-would-i-want-to-use-one

Reply:
I'm not against it, but I haven't learned how to use it yet.

Reply:
If you are using Android Studio on Linux, the default shortcut might be `Ctrl + Alt + L`.
You can find it here
![Format code](https://github.com/SCratORS/Everchanging/assets/6018890/cb4dae56-59cb-4107-aec2-a0ca33582a7e)

And you can change the shortcut here
[
![Formatter shortcut](https://github.com/SCratORS/Everchanging/assets/6018890/bf4403bc-ecaa-453e-b520-1bcc93015f59)
](url)

I also recommend running "Optimize imports". It removes unused imports and rearranges them alphabetically.
![Optimize imports](https://github.com/SCratORS/Everchanging/assets/6018890/35491867-6e9e-49e4-94d0-32d6b62032e4)

Reply:
I use Android Studio on Windows. Yes, it's in the same place. I will use it. Thanks for the hint